virtio-blk/src/request.rs

```diff
@@ -130,6 +130,22 @@ pub struct Request {
 unsafe impl ByteValued for RequestHeader {}
 
 impl Request {
+    /// Manually create mock `Request` instances.
+    #[cfg(test)]
+    pub fn new(
```

Comment:
What warning are we fixing with this patch?

For the future I suggest reporting the warning in the commit message, it's helpful to understand how to improve future code as well.

Reply:
The move was not lint-related, the method triggered the `missing_docs` lint and I noticed the method wasn't called in the `requests.rs` `tests` module, but from other files. Should I move it back or mention this in the commit message, you think?

Reply:
I would leave it in the test, I'm not familiar with this code, but maybe the idea was to instantiate it that way only in the test and not expose it to the user. It's true that it's conditional on the test here, but maybe it can be misinterpreted and accidentally enabled in the future.

In summary, I would make as little change as possible for now :-)

Reply:
Will do, thanks! :)

Reply:
Done